test/CMakeLists.txt

```
@@ -147,6 +152,4 @@ endif()
 set_property(TARGET ${PROJECT_NAME} APPEND_STRING PROPERTY COMPILE_FLAGS ${GODOT_COMPILE_FLAGS})
 set_property(TARGET ${PROJECT_NAME} APPEND_STRING PROPERTY LINK_FLAGS ${GODOT_LINKER_FLAGS})
 
-set_property(TARGET ${PROJECT_NAME} PROPERTY OUTPUT_NAME "gdexample")
-
-
+set_property(TARGET ${PROJECT_NAME} PROPERTY OUTPUT_NAME "gdexample")
```

Add newline at end of file to prevent warnings on GitHub:

```suggestion
set_property(TARGET ${PROJECT_NAME} PROPERTY OUTPUT_NAME "gdexample")

```